deploy.sh

```diff
@@ -1,62 +1,28 @@
 #!/bin/bash
-set -euxo pipefail # make bash quit if something weird happens
+set -euo pipefail # make bash quit if something weird happens
 
 export KUBE_NAMESPACE=${ENVIRONMENT}
-export KUBE_SERVER=${KUBE_SERVER}
+export KUBE_TOKEN=${KUBE_TOKEN}
+export VERSION=${VERSION}
 
-if [[ -z ${VERSION} ]] ; then
-    export VERSION=${IMAGE_VERSION}
-fi
+# passed to keycloak-gatekeeper and nginx for various proxy timeouts
+# the default is 60 seconds but audit has long-running queries
+export PROXY_TIMEOUT="300"
```

Comment:
Make it overrideable:
```suggestion
export PROXY_TIMEOUT=${PROXY_TIMEOUT:=300}
```
(not sure I got the syntax right. I guess really this should be a configmap)

Reply:
`:=` assigns to PROXY_TIMEOUT within itself and useful for `FOO=${BAR:=test}`
`:-` works well as we are exporting the same variable anyway as that exports 300 but doesn't set the default.

Reply:
Resolved 👍🏼